### PR TITLE
Migrations have been added and TestContainer has been configured

### PR DESCRIPTION
**Pull request by EnKarin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -94,7 +91,27 @@
...
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>1.17.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>1.17.2</version>
```
Comment (mfvanek) [inline pom.xml:103]: It's better to introduce a property to hold this version and use the property across all testcontainers dependencies

File: src/test/java/io/github/enkarin/bookcrossing/BookCrossingBaseTests.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            .withPassword("root")
+            .withDatabaseName("bookcrossing");
+
+    @DynamicPropertySource
+    public static void configure(final DynamicPropertyRegistry registry) {
+        registry.add("spring.datasource.url", MY_SQL_CONTAINER::getJdbcUrl);
+        registry.add("flyway.user", MY_SQL_CONTAINER::getUsername);
+        registry.add("flyway.password", MY_SQL_CONTAINER::getPassword);
+    }
+}
```
Comment (mfvanek) [inline src/test/java/io/github/enkarin/bookcrossing/BookCrossingBaseTests.java:28]: Add a simple test method in order to ensure that spring context loads

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: src/test/java/io/github/enkarin/bookcrossing/BookCrossingBaseTests.java (hunk 2 above)
Comment (EnKarin) [reply src/test/java/io/github/enkarin/bookcrossing/BookCrossingBaseTests.java:34]: Added

File: pom.xml (hunk 1 above)
Comment (EnKarin) [reply pom.xml]: Added

**Review by mfvanek — APPROVED** [review]

**Pull request merged**